src/CoreWCF.MSMQ/src/CoreWCF/Channels/MsmqQueueTransport.cs

```diff
@@ -21,6 +21,7 @@ public class MsmqQueueTransport : IQueueTransport, IDisposable
         private readonly TimeSpan _queueReceiveTimeOut;
         private readonly IDeadLetterQueueMsmqSender _deadLetterQueueSender;
         private readonly ILogger<MsmqQueueTransport> _logger;
+        public int ConcurrencyLevel => 1;
 
```

Comment:
Properties come after the constructor. When I first looked at this I thought you had made a public field because of where it's located in the source file. A little confusing having it at the top.